src/unix/linux_like/android/mod.rs

```diff
@@ -638,6 +643,15 @@ s_no_extra_traits! {
         pub ifc_len: c_int,
         pub ifc_ifcu: __c_anonymous_ifc_ifcu,
     }
+
+    // linux/if_ether.h
+
+    #[repr(C, align(1))]
```

Comment:
This struct is [packed](https://cs.android.com/android/platform/superproject/main/+/main:bionic/libc/kernel/uapi/linux/if_ether.h;l=128;bpv=1;bpt=1) and not align-1.

Reply:
I know. was trying to solve few architectures build failures.